[2025-03-31T13:54:09.643+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-31T13:54:09.649+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: check_dag.read_file manual__2025-03-31T08:23:57.308511+00:00 [queued]>
[2025-03-31T13:54:09.654+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: check_dag.read_file manual__2025-03-31T08:23:57.308511+00:00 [queued]>
[2025-03-31T13:54:09.655+0530] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-03-31T13:54:09.664+0530] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): read_file> on 2025-03-31 08:23:57.308511+00:00
[2025-03-31T13:54:09.667+0530] {standard_task_runner.py:72} INFO - Started process 20770 to run task
[2025-03-31T13:54:09.669+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'check_dag', 'read_file', 'manual__2025-03-31T08:23:57.308511+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/check_dag.py', '--cfg-path', '/tmp/tmpn5rgbanj']
[2025-03-31T13:54:09.671+0530] {standard_task_runner.py:105} INFO - Job 22: Subtask read_file
[2025-03-31T13:54:09.698+0530] {task_command.py:467} INFO - Running <TaskInstance: check_dag.read_file manual__2025-03-31T08:23:57.308511+00:00 [running]> on host Voldemort.
[2025-03-31T13:54:09.749+0530] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='check_dag' AIRFLOW_CTX_TASK_ID='read_file' AIRFLOW_CTX_EXECUTION_DATE='2025-03-31T08:23:57.308511+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-31T08:23:57.308511+00:00'
[2025-03-31T13:54:09.750+0530] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-31T13:54:09.758+0530] {logging_mixin.py:190} INFO - b'Hi there!\n'
[2025-03-31T13:54:09.758+0530] {python.py:240} INFO - Done. Returned value was: None
[2025-03-31T13:54:09.762+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-31T13:54:09.762+0530] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=check_dag, task_id=read_file, run_id=manual__2025-03-31T08:23:57.308511+00:00, execution_date=20250331T082357, start_date=20250331T082409, end_date=20250331T082409
[2025-03-31T13:54:09.802+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-31T13:54:09.811+0530] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-31T13:54:09.812+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
